app/components/PasswordField/PasswordField.jsx

```diff
@@ -14,6 +20,19 @@ class PasswordField extends Component<Props, State> {
     showKey: false
   }
 
+  constructor () {
+    super()
+    this._handleKeyPress = this._handleKeyPress.bind(this)
```

Comment:
You can use arrow functions instead of binding.

So just have a `handleKeyPress = (target) => { }` (without the constructor)

Reply:
hey, good tip! thanks